app/models/marc_fields/marc_field_wrapper.rb

```diff
@@ -0,0 +1,49 @@
+##
+# This class is intended to wrap Marc::DataFields that come from the marc gem.
+# This wrapper class is used to enhance the base Marc::DataField with additional behavior
+# such as parsing and retaining encoded data that might otherwise be removed for display (e.g. $6)
+# as well as add an attribute writer so subfields can be set as well as read.
```

Comment:
Nope, we wrap all fields in this wrapper in the base class.  Both `880` and the regular MARC fields they map to have a $6, and this gives us access to that data even after it has been removed from display (which may or may not be required by some subclasses of `MarcField`).